spec/support/dummy/schema.rb

```diff
@@ -394,7 +394,7 @@ def call(obj, args, ctx)
       argument :input, !ReplaceValuesInputType
       resolve ->(o, args, ctx) {
         GLOBAL_VALUES.clear
-        GLOBAL_VALUES.push(*args[:input][:values])
+        GLOBAL_VALUES.push(args.input.values)
```

Comment:
(I think the specs are failing because we lost a `*` here)

Reply:
Aha! Good seeing!